projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -956,8 +963,8 @@ public function render_widget_edit_filter( $filter, $is_template = false, $is_in
 
 		$args['name_placeholder'] = Helper::generate_widget_filter_name( $args );
 
-		// Hide author filter when Instant Search is turned off.
-		if ( ! $is_instant_search && 'author' === $args['type'] ) :
+		// Hide author & blog ID filters when Instant Search is turned off.
+		if ( ! $is_instant_search && ( 'author' === $args['type'] || 'blog_id' === $args['type'] ) ) :
```

Comment:
I suggest checking the type with `in_array`, which would be more readable.

```suggestion
		if ( ! $is_instant_search && in_array( $args['type'], array( 'author', 'blog_id' ) ) ) :
```

Reply:
@dognose24 Thank you for the suggestion, I have refactored it.